### DISCUSSION TITLE
Is "Jump to Dashboard" feature released?

### DISCUSSION THREAD

**Discussion by jkryanchou** [apache/superset#29363] [Ideas]
I have seen the video https://www.youtube.com/watch?v=GrlRuOELJRw show the "Jump to Dashboard" feature. While I found nothing on it in document or Github repo. Is it released? Or still in development.  
![image](https://github.com/apache/superset/assets/1206342/6e607e2d-f143-46da-803e-5280537f56a3)

**Comment by rusackas** [comment]
Nope. This is/was a feature in development, but it has not yet been finalized or opened as a PR. @cccs-tom might know if this is still on their roadmap.

**Reply by cccs-tom** [reply]
Unfortunately, our team has come to the realization that the data exploration workflows our users are asking us to enable are simply too different from the core workflows available in Superset. `Jump to Dashboard` was meant to be one step toward fulfilling their data exploration requirements. As it stands, we are re-focusing our Superset efforts on true BI use cases (which we also have) and we have now pivoted to designing a custom data exploration tool that is tailored to their needs.

**Reply by jkryanchou** [reply]
While this feature is really needed. I thought that I could interacted with each dashboard by "Jump to Dashboard". I have seen the Metabase has already support this feature for a long time.
Could we archive this by other means in latest version?

**Reply by jkryanchou** [reply]
Or you could provide the feature like we could customize the click destination as what metabase did https://www.metabase.com/learn/dashboards/custom-destinations
